src/app/components/impersonification-notification/impersonification-notification.coffee

```diff
@@ -0,0 +1,49 @@
+ImpersonificationNotificationCtrl = ($scope, $log, toastr, MnoeUserAccessRequests) ->
+  'ngInject'
+
+  $scope.isLoading = true
+
+  $scope.initialize = (requests) ->
+    $scope.requests = requests
+    $scope.isLoading = false
+
+  $scope.requesterName = (request) ->
+    request.requester.name + ' ' + request.requester.surname
+
+  $scope.denyAccess = (index, request) ->
+    $scope.isLoading = true
+    MnoeUserAccessRequests.deny(request.id).then(
+      () ->
+        $scope.requests.splice(index, 1)
+        toastr.success('mno_enterprise.templates.impac.impersonification_notification.deny.success_toastr', {extraData: {name: $scope.requesterName(request)}})
+      (errors) ->
+        $log.error(errors)
+        toastr.error('mno_enterprise.templates.impac.impersonification_notification.deny.error_toastr', {extraData: {name: $scope.requesterName(request)}})
+    ).finally(-> $scope.isLoading = false)
+
+  $scope.approveAccess = (index, request) ->
```

Comment:
`request` is the functional object here, the arguments order should be switched.